src/styles.scss

```
@@ -250,7 +251,7 @@ $green-font-color: #19B028;
 // Custom border classes.
 .border-color-grey { border-color: var(--border) !important; }
 .border-color-blue { border-color: $buy-button-blue; }
-.tab-underline-active { border-bottom: solid $buy-button-blue 3px; }
+.tab-underline-active { border-bottom: solid $normal-font-color 3px; }
```

```suggestion
.tab-underline-active { border-bottom: solid $buy-button-blue 3px; }
```

this works better on non-light themes